crates/server/src/main.rs

```diff
@@ -38,8 +43,29 @@ async fn main() -> Result<()> {
     args.init_tracing();
     tracing::debug!("Args: {args:?}");
 
-    let demo_operator_key = std::env::var("WARG_DEMO_OPERATOR_KEY")?;
-    let signing_key = demo_operator_key.parse()?;
+    // TODO(#46): Add --warg-demo-operator-key[-file] and others cli options
```

Comment:
This looks like it could be its own function called e.g. `get_operator_key`

Reply:
Yes, I expected something like this would be needed soon. I'm not familiar with what's available in Rust libraries in general yet. I'd expect it perhaps be a bit more generic where selected props come from either a command line argument or an environment variable in that order, with preference given to a "file" variation if set for a limited set of things like passwords, etc.